Sections/US-States/FL/GPP Extension: IAB Privacy’s Florida Privacy Technical Specification.md

```diff
@@ -0,0 +1,190 @@
+          <span style="color:rgb(36, 41, 47);">(5). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Citizenship or Immigration Status.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(6). Consent to Process the Consumer’s Sensitive Data Consisting of Genetic Data for the Purpose of Uniquely Identifying an Individual.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(7). Consent to Process the Consumer’s Sensitive Data Consisting of Biometric Data for the Purpose of Uniquely Identifying an Individual.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(8). Consent to Process the Consumer’s Sensitive Data Consisting of Precise Geolocation Data.</span>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">KnownChildSensitiveDataConsents</span>
+        </td>
+        <td>Bitfield(6)</td>
+        <td>Two bits for each Data Activity:<p></p><code>0</code> = Not Applicable, the Controller does not
+          <span style="color:rgb(36, 41, 47);">Process Sensitive Data of a known Child</span><p></p><code>1</code> = Consent <p></p><code>2</code> = No Consent&nbsp;<span style="color:rgb(36, 41, 47);"><p></p>(1). Consent to Process Sensitive Data from a Known Child Under the Age of 13.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(2). Consent to Process Sensitive Data of Consumers At Least 13 Years of Age but Younger Than 16 Years of Age.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(3). Consent to Process the Sensitive Data of Consumers At Least 16 Years of Age but Younger Than 18 Years of Age.</span><p></p>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">AdditionalDataProcessingConsent</span>
+        </td>
+        <td>Int(2)</td>
+        <td>Consent to Processing of the Consumer’s Personal Data that Is Not Reasonably Necessary for nor Compatible with the Disclosed Purpose(s) for which the Consumer’s Personal Data Was Processed<p></p><code>0</code> = Not Applicable, the Controller does not Process Personal Data that is Not Reasonably Necessary for nor Compatible with the Disclosed Purpose(s)<p></p><code>1</code> = Consent <p></p><code>2</code> = No Consent&nbsp;</td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">MspaCoveredTransaction</span>
+        </td>
+        <td>Int(2)</td>
+        <td>
+          <span style="color:rgb(36, 41, 47);">Publisher or Advertiser, as applicable, is a signatory to the IAB Multistate Service Provider Agreement (MSPA), as may be amended from time to time, and declares that the transaction is a “Covered Transaction” as defined in the MSPA.&nbsp;</span><p></p><code>0</code> = Not Applicable<p></p><code>1</code> = Yes<p></p><code>2</code> = No</td>
```

Comment:
N/A is not a choice for covered on other strings

Reply:
We can update after public comment so that covered transaction is:
1 Yes

2 No

This will also be done in MT, OR, TX sections.

Reply:
thanks!